docs/get-started/README.md

```diff
@@ -1,9 +1,30 @@
 # Getting Started
 
-Welcome to luma.gl!
+To use luma.gl in an application, simply install it using `yarn`:
 
-This section has information about:
+```
+yarn add luma.gl
+```
 
-* how to install luma.gl
-* how to run, copy, and modify the many luma.gl examples
-* How to use luma.gl with other frameworks in different environments.
+Note: While we'll use `yarn` for these instructions, as it's the tool we use for development, `npm` can be used as a drop-in substitute in most cases. A map of `npm` instructions to `yarn` is available [here](https://yarnpkg.com/lang/en/docs/migrating-from-npm/).
+
+## Using with Node.js
+
+luma.gl is built to run using [headless-gl](https://www.npmjs.com/package/gl) under Node.js, which can be extremely useful for unit testing. It is important to note that `headless-gl` only supports WebGL 1 and few extensions, so not all of luma.gl's features will be available.
+
+Use `yarn install gl` to install `headless-gl`. luma.gl will automatically use it when running under Node.js. You can then create a context using the `createGLContext` context function.
+
+```js
+import 'luma.gl';
+import {createGLContext, Model, ...} from '@luma.gl/core';
+const gl = createGLContext({width, height, ...});
+```
+
+luma.gl's I/O functions (e.g. `loadImage`) are also written to work both in the browser and under Node.js.
```

Comment:
Cut, IO has been moved to loaders.gl

Reply:
Chopping block? https://github.com/uber/luma.gl/blob/master/modules/webgl/src/utils/load-file.js
:)

Reply:
The loadImage function is used by the async textures so it is needed at least internally. 

It technically does not need to be exported from luma.gl, however website relies on being able to control path prefix in examples so there is a wrinkle that would have to be solved.